mdaviz/chartview.py

```diff
@@ -155,7 +155,7 @@ def addCurve(self, row, *args, **kwargs):
         # Add to the dictionary
         label = kwargs.get("label", None)
         self.line2D[label] = plot_obj[0], args[0], args[1], row
-        print(f"Calling addCurve: row={self.line2D[label][3]}")
+        # print(f"Calling addCurve: row={self.line2D[label][3]}")
```

Comment:
There are a lot of print statements commented out, do we need them?

Reply:
probably not, let me removed them

Reply:
Done